### PR TITLE
Add tests for selinux labels

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
bot, retest this please

**Comment by TomSweeneyRedHat** [comment]
LGTM and happy green test buttons

**Comment by umohnani8** [comment]
@rhatdan can you add a test for privileged vs unprivileged. Check that the mount label is set even when privileged.

**Comment by rhatdan** [comment]
@umohnani8 now there is some --priviliged tests.

**Comment by umohnani8** [comment]
@rhatdan thanks! LGTM

File: test/e2e/run_selinux_test.go [hunk 1]
```diff
@@ -84,4 +84,68 @@ var _ = Describe("Podman run", func() {
...
+	It("podman test selinux label /run/secrets", func() {
+		session := podmanTest.Podman([]string{"run", fedoraMinimal, "ls", "-dZ", "/run/secrets"})
+		session.WaitWithDefaultTimeout()
+		Expect(session.ExitCode()).To(Equal(0))
+		match, _ := session.GrepString("container_file_t")
+		Expect(match).Should(BeTrue())
+	})
+
+	It("podman test selinux --privilged label resolv.conf", func() {
+		session := podmanTest.Podman([]string{"run", "--privilged", fedoraMinimal, "ls", "-Z", "/etc/resolv.conf"})
```
Comment (mheon) [inline test/e2e/run_selinux_test.go]: Typo privilged -> privileged (and for the next 3 tests too)

**Comment by mheon** [comment]
One typo, LGTM once fixed

**Comment by rhatdan** [comment]
@umohnani8 I see all tests for --privileged failing.
Did you make mountlabel work with the --privileged flag?  I don't see it locally.

```
./bin/podman run --privileged -ti fedora ls -Z /
        unconfined_u:object_r:container_share_t:s0 bin
        unconfined_u:object_r:container_share_t:s0 boot
unconfined_u:object_r:container_runtime_tmpfs_t:s0 dev
        unconfined_u:object_r:container_share_t:s0 etc
        unconfined_u:object_r:container_share_t:s0 home
        unconfined_u:object_r:container_share_t:s0 lib
        unconfined_u:object_r:container_share_t:s0 lib64
        unconfined_u:object_r:container_share_t:s0 lost+found
        unconfined_u:object_r:container_share_t:s0 media
        unconfined_u:object_r:container_share_t:s0 mnt
        unconfined_u:object_r:container_share_t:s0 opt
                       system_u:object_r:proc_t:s0 proc
        unconfined_u:object_r:container_share_t:s0 root
        unconfined_u:object_r:container_share_t:s0 run
        unconfined_u:object_r:container_share_t:s0 sbin
        unconfined_u:object_r:container_share_t:s0 srv
                      system_u:object_r:sysfs_t:s0 sys
        unconfined_u:object_r:container_share_t:s0 tmp
        unconfined_u:object_r:container_share_t:s0 usr
        unconfined_u:object_r:container_share_t:s0 var
```

**Comment by rhatdan** [comment]
@umohnani8 Remind me why we want these labels set?

**Comment by umohnani8** [comment]
@rhatdan because that is the way it is in docker and cri-o. The mount label is set even in privileged mode.

**Comment by rhatdan** [comment]
@mheon @baude @umohnani8 @TomSweeneyRedHat @vrothberg I think this is ready to merge.  Passes all tests.

**Comment by TomSweeneyRedHat** [comment]
LGTM and all happy green test buttons.

**Comment by umohnani8** [comment]
LGTM

**Pull request merged**